CHANGELOG.md

```diff
@@ -27,8 +27,8 @@ COMPATIBILITY:
 
 CONTAINER IMAGES:
 
-- Control plane: `ghcr.io/nginxinc/nginx-gateway-fabric:1.5.1`
-- Data plane: `ghcr.io/nginxinc/nginx-gateway-fabric/nginx:1.5.1`
+- Control plane: `ghcr.io/nginx/nginx-gateway-fabric:1.5.1`
```

Comment:
I only pushed `1.5.1` to the packages here. Not sure if we want to add all the old ones.